packages/edit-site/src/components/page-pages/index.js

```diff
@@ -75,6 +76,20 @@ export default function PagePages() {
 
 	const fields = useMemo(
 		() => [
+			{
+				id: 'featured-image',
+				header: __( 'Featured Image' ),
+				accessorFn: ( page ) => page.featured_media,
+				cell: ( props ) =>
+					!! props.row.original.featured_media ? (
```

Comment:
Still picking nits with `!!`: this cast-to-boolean inside the ternary condition is unnecessary. :)

Reply:
I like these when the props are not booleans 😆